cime/config/e3sm/config_grids.xml

```diff
@@ -1827,6 +1867,20 @@
       <file grid="ice|ocn" mask="oQU240">domain.ocn.ne4pg2_oQU240.190321.nc</file>
       <desc>ne4np4.pg2 is Spectral Elem 7.5-deg grid w/ 2x2 FV physics grid per element:</desc>
     </domain>
+    <domain name="ne4np4.pg3">
+      <nx>864</nx>
+      <ny>1</ny>
+      <file grid="atm|lnd" mask="oQU240">?????</file>
```

Comment:
?????

Reply:
yea, I haven't generated those files. This is mainly just a placeholder, but it will fail if someone tries to run it. I plan to fill these out, but it just takes work. I can remove them if you'd prefer.